test-framework/core/src/main/java/org/keycloak/testframework/server/KeycloakServerConfigBuilder.java

```diff
@@ -70,6 +73,11 @@ public KeycloakServerConfigBuilder dependency(String groupId, String artifactId)
         return this;
     }
 
+    public KeycloakServerConfigBuilder configFile(Path filePath) {
```

Comment:
Would it be better to just add `String resource` here instead? Much simpler to use then than having to resolve a file path in the test itself.

Reply:
Yes makes a lot of sense